files/tasks.py

```
@@ -491,10 +517,11 @@ def encode_media(
             encoding.save(
                 update_fields=["status", "logs", "progress", "total_run_time"]
             )
-        # this will raise a django.db.utils.DatabaseError error when task is revoked,
-        # since we delete the encoding at that stage
-        except:
-            pass
+        except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)